src/sass/messaging/partials/_message-compose.scss

```diff
@@ -50,18 +56,36 @@
 
 .messaging-recipient > div,
 .messaging-from,
-.messaging-subject-group {
+.messaging-subject-field {
   padding: 1.5rem;
+
+  @media (min-width: $medium-screen) {
+    padding-top: .75rem;
+    padding-bottom: .75rem;
+  }
+}
+
+.messaging-subject-field legend {
+  // Should match padding above;
+  @media (max-width: $medium-screen) {
+    padding-top: 1.5rem;
+  }
 }
 
 .messaging-from,
 .messaging-recipient,
-.messaging-subject-group {
+.messaging-subject-field {
   @media (max-width: $medium-screen) {
     border-bottom: 1px solid $color-gray;
   }
 }
 
+[id=messaging-compose] {
```

Comment:
I try to avoid high-specificity selectors because they can lead to bloat. Probably not a huge deal here because it's relatively isolated. Reluctant to change it.